sql/core/src/main/scala/org/apache/spark/sql/execution/window/BoundOrdering.scala

```diff
@@ -24,14 +24,14 @@ import org.apache.spark.sql.catalyst.expressions.Projection
 /**
  * Function for comparing boundary values.
  */
-private[window] abstract class BoundOrdering {
+private[sql] abstract class BoundOrdering {
```

Comment:
BTW, why is this `[sql]`? Looks it can be `[window]`.

Reply:
Yeah good catch. Reverted. I guess I was accessing this in WindowInPandasExec before